src/Symfony/Component/Config/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.1.0
+-----
+
+ * Added `allowEmptyValue` method to `NumericNodeDefinition` and `BooleanNodeDefinition` to accept `null` values.
```

Comment:
what about naming this `allowNull`?

Reply:
I'm fine with it as it particularly makes sense for these noce types, but do we care this would internally use the `allowEmptyValue`  mechanism, or should we add another mechanic for this?
First option seems relevant enough to me.

Reply:
I would ensure the public API is as understandable as possible. `allowNull` looks better than `allowEmptyValue` if that's what it means (I don't know if `allowEmptyValue` means `allowNull` or any falsy value)